monai/transforms/spatial/array.py

```diff
@@ -460,12 +473,7 @@ def __call__(
         scale_extent = self.scale_extent if scale_extent is None else scale_extent
         output_shape, offset = compute_shape_offset(data_array.shape[1:], affine_, new_affine, scale_extent)
```

Comment:
Does the shape here `data_array.shape[1:]` be the pending shape or it will compute the output shape based on the original shape?

Reply:
you are right, this should be the pending shape